src/main/scala/com/microsoft/partnercatalyst/fortis/spark/streamfactories/RedditStreamFactory.scala

```diff
@@ -0,0 +1,27 @@
+package com.microsoft.partnercatalyst.fortis.spark.streamfactories

```

Comment:
General comment: I wonder if we want to move all the `streamXYZ` packages into a single wrapper package called `sources`. In that case, we'd have sources, transforms and sinks as our top-level abstractions in the project which is a fairly standard vocabulary.